crates/cheatcodes/tests/assume.rs

```diff
@@ -0,0 +1,90 @@
+//! Contains various tests for `assume*` cheatcodes.
+
+use alloy_primitives::U256;
+use foundry_config::{Config, FuzzConfig};
+use foundry_test_utils::{forgetest_init, str};
+
+forgetest_init!(test_assume_no_revert, |prj, cmd| {
```

Comment:
I would like these tests to be all in `forge`

Reply:
yep, will move them all under `forge/tests/it` then

Reply:
I reverted the change, indeed they should stay in cheats testdata, got 
Will follow up with a PR to cleanup `force/cli/test_cmd.rs` as it is now a catch all thingy